### PR TITLE
[KAFKA-6485] 'ConsumerGroupCommand' performance optimization for old consumer describe group

### PR DESCRIPTION
**Pull request by leonhong**
https://issues.apache.org/jira/browse/KAFKA-6485

ConsumerGroupCommand describegroup performance optimization.
performance improvement 3 times compare trunk(1.0+). and performance improvement 10 times compare 0.10.2.1

`./kafka-consumer-groups.sh --zookeeper 127.0.0.1 --group cp-mirror-consumer-group --describe
`
below is truck version:
1ada9efa5-850a-4495-83b6-9a28f8885512

below is  performance improvement 3 times:
75a51a9f-d2c3-47fa-81d3-519343aa4d84

below is  performance improvement 10 times for 0.10.2.1:
3a89bcef-ac5f-4383-94ae-36897ae2da26

### CODE REVIEW

**Comment by rhauch** [comment]
@leonhong, please change the title of this PR to begin with `KAFKA-6485: 'ConsumerGroupCommand' ...` so that the bot recognizes the pattern correctly and automatically associates this PR with the issue.

**Comment by huxihx** [comment]
LGTM although I am not a committer :-)

File: core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala [hunk 1]
```diff
@@ -443,9 +445,9 @@ object ConsumerGroupCommand extends Logging {
     private def getPartitionOffsets(group: String,
                                     topicPartitions: Seq[TopicPartition],
                                     channelSocketTimeoutMs: Int,
```
Comment (hachikuji) [inline core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala:447]: Now `channelSocketTimeoutMs` and `channelRetryBackoffMs` are unused.

**Review by hachikuji — COMMENTED** [review]
Seems reasonable. Left one comment.

**Comment by hachikuji** [comment]
Closing this since the old consumer has been removed from the codebase.

**Pull request closed without merging**